github-runner-manager/src/github_runner_manager/configuration/base.py

```diff
@@ -91,6 +94,8 @@ class SupportServiceConfig(BaseModel):
     proxy_config: "ProxyConfig | None"
     runner_proxy_config: "ProxyConfig | None"
     use_aproxy: bool
+    aproxy_exclude_addresses: str | None
+    aproxy_redirect_ports: str | None
```

Comment:
Would it make sense to model this as a python list? For me, the value should be independent of the output format.